packages/backend/server/routes/publish.post.ts

```diff
@@ -7,19 +7,22 @@ import { randomUUID } from "uncrypto";
 import { setItemStream, useTemplatesBucket } from "~/utils/bucket";
 import { useOctokitInstallation } from "~/utils/octokit";
 import { generateTemplateHtml } from "~/utils/template";
+import type { PackageManager } from "@pkg-pr-new/utils";
```

Comment:
If this type is only used in this file, then let's inline it here and remove it from utils!